kubernetes/services/sg_manager.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sg-configmap
+data:
+  application.properties: |
+    ignite.host=10.99.135.32
+    ignite.port=10800
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: sgmanager
+  name: sgmanager
+spec:
+  selector:
+    matchLabels:
+      app: sgmanager
+  replicas: 1
+  #replicas: 3
+  template:
+    metadata:
+      labels:
+        app: sgmanager
+    spec:
+      volumes:
+        - name: sg-volume
+          configMap:
+            name: sg-configmap
+            items:
+              - key: application.properties
+                path: application.properties
+      containers:
+        - image: zhonghaolyu/repo:sgKube1
+        #- image: fwnetworking/controller:node_manager-v0.3.0
+          name: sgmanager-web
+          imagePullPolicy: IfNotPresent
```

Comment:
How big is the image? We could want to change the policy to alwaysPull.

Reply:
About several hundreds MB